runtime/oti/j9nonbuilder.h

```diff
@@ -5460,6 +5460,7 @@ typedef struct J9JavaVM {
 #ifdef J9VM_OPT_VALHALLA_VALUE_TYPES
 	UDATA valueFlatteningThreshold;
 #endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
+	UDATA deepScanEnabled;
```

Comment:
Can you check if there is a free bit in the `runtimeFlags`, `extendedRuntimeFlags `, or `extendedRuntimeFlags2`>

Reply:
👍 Added to `extendedRuntimeFlags2`